dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.40/V2_40_12__Create_table_program_stage_working_list.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE if not exists programstageworkinglist (
+    programstageworkinglistid integer NOT NULL,
```

Comment:
should this maybe be a primary key https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-PRIMARY-KEYS

Reply:
We use `constraint primary key` as part of the constraints further down, which is missing.